web/src/engine/websites/Baozimh.ts

```diff
@@ -0,0 +1,45 @@
+import { Tags } from '../Tags';
+import icon from './Baozimh.webp';
+import { type Chapter, DecoratableMangaScraper, Page } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchHTML } from '../platform/FetchProvider';
+
+function ChapterInfoExtractor(anchor: HTMLAnchorElement) {
+    return {
+        id: anchor.pathname + anchor.search,
+        title: anchor.text.trim()
+    };
+}
+
+@Common.MangaCSS(/^{origin}\/comic\/[^/]+/, '.comics-detail__title')
+@Common.MangasNotSupported()
```

Comment:
I think there is a useable list
https://www.baozimh.com/api/bzmhq/amp_comic_list?type=all&region=all&filter=*&page=1&limit=50

Reply:
Nice catch !